### PR TITLE
(DOCSP-27337): Backend release notes for Jan 11, 2023

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Backend release notes for Jan 11, 2023

### Jira

- https://jira.mongodb.org/browse/DOCSP-27337

### Staged Changes

- [Backend release notes ](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-27337/release-notes/backend)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/release-notes/backend.txt [hunk 1]
```diff
@@ -8,6 +8,18 @@ Atlas App Services Changelog
...
+
+2023 Releases
+-------------
+
+.. _backend_20230111:
+
+11 January 2023 Release
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Configure user creation function via the App Services CLI and Admin API.
```
Comment (mongodben) [inline source/release-notes/backend.txt:21]: TODO: add ref on merge of changes for https://jira.mongodb.org/browse/DOCSP-26904

Comment (dacharyc) [reply source/release-notes/backend.txt:21]: It looks like [that related PR](https://github.com/mongodb/docs-app-services/pull/301) got merged - do you wanna add a link?

**Review by dacharyc — APPROVED** [review]
LGTM but I think the related PR is merged now and you could probably add that link listed in your TODO here.

File: source/release-notes/backend.txt (hunk 1 above)
Comment (mongodben) [reply source/release-notes/backend.txt:21]:
```suggestion
- Configure :ref:`user creation function <user-creation-function>` via the App Services CLI and Admin API.
```

**Pull request merged**